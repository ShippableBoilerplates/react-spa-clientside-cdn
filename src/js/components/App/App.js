import React, {PureComponent} from 'react'
import Header from '../Header/Header'
import Clicker from '../Clicker/Clicker'

import {css} from 'glamor'
import 'glamor/reset'
import {wrapper} from './App.styles'

css.global(
    'html', {
        fontFamily: 'Roboto, sans-serif',
        fontWeight: '400'
    }
)

export default class App extends PureComponent {
    render() {
        return (
            <div {...wrapper}>
                <Header />
                <Clicker />
            </div>
        )
    }
}
