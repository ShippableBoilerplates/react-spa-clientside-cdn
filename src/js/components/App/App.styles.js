import {css} from 'glamor'

export const wrapper = css({
    height: '100vh',
    background: '#DDDDDD'
})
