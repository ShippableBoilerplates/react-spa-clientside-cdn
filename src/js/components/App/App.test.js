import React from 'React'
import {shallow} from 'enzyme'
import App from './App'

test('should render Header correctly', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
})
