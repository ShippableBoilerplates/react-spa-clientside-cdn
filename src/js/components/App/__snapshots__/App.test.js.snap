// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  data-css-1ak43c1=""
>
  <Header />
  <Connect(Clicker) />
</div>
`;
